src/components/PupilViews/PupilReview/PupilReview.view.tsx

```diff
@@ -28,6 +33,12 @@ type PupilViewsReviewProps = {
   phase?: "primary" | "secondary";
   starterQuizQuestionsArray: QuestionsArray;
   exitQuizQuestionsArray: QuestionsArray;
+  programmeSlug: string;
```

Comment:
As per my comment on pupil experience, lets just use browse data and pageType instead of all of these individual props.